locales/en/messages.json

```diff
@@ -3112,6 +3128,72 @@
+    },
+    "pidTuningDTermFilterSlider": {
+        "message": "D Term Filter Multiplier:"
+    },
+    "pidTuningDTermFilterSliderHelp": {
+        "message": "Raises or Lower the default D-term Lowpass Filters in proportion to each-other.<br>The D-term filtering is applied after the PID loop, ONLY on the D-term, since it is the term most sensitive to noise and can amplify any high frequency noise by 10x to 100x plus.<br><br>Generally you want to move the D-term Filter slider up with the Gyro Filter slider. You want to have the D-term filter cutoffs well below the motor noise range for harder D-term filtering then what is applied on the entire gyro signal with the Gyro Filter Multiplier.<br>That recommended differential is built into the default and slider scaling."
+    },
+    "pidTuningPidSlidersHelp": {
+        "message": "Sliders to adjust the quad flight characteristics (PID gains)<br><br>Master Multiplier: Raises or Lowers all the PID gains (above) holding the proportional difference between the gains.<br><br>PD Balance: Adjusts the balance (ratio \\ proportional difference) between the P and D terms ('the spring' [p-term] and 'shock absorber' [d-term]).<br><br>PD Gain: Raises or Lowers the P&D gains together - holding the ratio (balance) between the two - for more (or less) PID control authority.<br><br>Stick Response Gain: Raises or Lowers the FeedForward gains to control the stick response feel of the quad."
+    },
+    "pidTuningSliderHighWarning": {
+        "message": "<span class=\"message-negative\">CAUTION</span>: Pushing sliders this high may cause flyaways, motor damage or unsafe craft behaviour. Please proceed with caution."
+    },
+    "pidTuningSlidersDisabled": {
+        "message": "<strong>Note:</strong> Sliders are disabled because values were changed manually. Moving the sliders will activate them again. This will reset the values and any unsaved changes will be lost."
+    },
+    "pidTuningSliderLow": {
+        "message": "Low"
+    },
+    "pidTuningSliderDefault": {
+        "message": "Default"
+    },
+    "pidTuningSliderHigh": {
+        "message": "High"
+    },
+    "pidTuningMasterSlider": {
+        "message": "Master Multiplier:"
+    },
+    "pidTuningPDRatioSlider": {
+        "message": "PD Balance:"
+    },
+    "pidTuningPDGainSlider": {
+        "message": "P and D Gain:"
+    },
+    "pidTuningResponseSlider": {
+        "message": "Stick Response Gain:"
+    },
+    "pidTuningMasterSliderHelp": {
+        "message": "Generally larger quads need higher PID gains due to having a lower power to weight ratio.<br>Smaller quads (micros) generally need lower PID gains due to a higher power to weight ratio.<br>Higher values are for lower authority quads. Lower values are for higher authority quads."
```

Comment:
Add a double `<br>` to be aligned with the others help messages. It will be great if you can add some `description` fields to the messages to help the translators.

Reply:
I will add descriptions. What do you mean by double `<br>`?

Reply:
The other similar help messages of this PR use `<br><br>` inside the text to separate paragraphs. This text has only one `<br>`.
![image](https://user-images.githubusercontent.com/2673520/64249339-b9cc1a00-cf13-11e9-8f0d-e069c1bbc202.png)
![image](https://user-images.githubusercontent.com/2673520/64249358-c6507280-cf13-11e9-948b-fd5252e2e5de.png)

Reply:
Yeah, I will do that. 👍